nixos/modules/services/web-apps/taguette.nix

```diff
@@ -0,0 +1,160 @@
+{
+  config,
+  lib,
+  pkgs,
+  options,
+  ...
+}:
+let
+  cfg = config.services.taguette;
+  opt = options.services.taguette;
+  settingsFormat = pkgs.formats.pythonVars { };
+in
+{
+  options = {
+    services.taguette = {
+      enable = lib.mkEnableOption "the Taguette server";
+      package = lib.mkPackageOption pkgs "taguette" { };
+
+      settings = lib.mkOption {
+        description = ''
+          Configuration options.
+        '';
+
+        defaultText = ''
+          NAME = "NixOS Taguette Server";
+        '';
+
+        default = {
+          NAME = lib.mkDefault "NixOS Taguette Server";
+          BIND_ADDRESS = lib.mkDefault cfg.host;
+          PORT = lib.mkDefault cfg.port;
+          BASE_PATH = lib.mkDefault "";
+          SECRET_KEY = lib.mkDefault (
+            if cfg.secretKeyFile != null then
+              ''
+                $(cat ${cfg.secretKeyFile})
+              ''
+            else
+              "{secret}"
+          );
```

Comment:
Unless I'm missing something, this is supposed to be using `pkgs.formats.pythonVars` to generate the file, shell substitution won't work.